packages/common/src/utils/analytics.ts

```diff
@@ -17,22 +19,31 @@ export const attachAnalyticsParams = (item: PlaylistItem) => {
 
   sources.map((source: Source) => {
     const url = new URL(source.file);
+    const urlSearchParams = url.searchParams;
 
     const mediaId = mediaid.toLowerCase();
     const sourceUrl = url.href.toLowerCase();
 
-    // Attach user_id and profile_id only for VOD and BCL SaaS Live Streams
-    const isVOD = sourceUrl === `https://cdn.jwplayer.com/manifests/${mediaId}.m3u8`;
-    const isBCL = sourceUrl === `https://content.jwplatform.com/live/broadcast/${mediaId}.m3u8`;
+    const cdnUrlVOD = `https://cdn.jwplayer.com/manifests/${mediaId}.m3u8`;
+    const cdnUrlBCL = `https://content.jwplatform.com/live/broadcast/${mediaId}.m3u8`;
```

Comment:
Do you think we should remove the extension check? The sources could also contain MPEG-DASH or MP4 files that won't have the analytic parameters.